Tasks/Common/utility-common/downloadutility.ts

```diff
@@ -2,13 +2,45 @@
 
 var https   = require('https');
 var fs      = require('fs');
+var url = require('url');
 import * as tl from "vsts-task-lib/task";
 
-export async function download(url: any, downloadPath: string, printData: boolean): Promise<void> {
+function isDownloadSucceeded(response: any): boolean {
+    return response.statusCode >= 200 && response.statusCode < 300;
+}
+
+function isRedirect(response: any): boolean {
+    return response.statusCode >= 300
+             && response.statusCode < 400
+             && response.headers
+             && response.headers.location;
+}
+
+function getRedirectOptions(options: any, redirectUrl: string): any {
+    tl.debug("Download latest release from redirect uri: " + redirectUrl);
```

Comment:
Change the debug log from Download latest release … to something more meaningful.